src/index.js

```diff
@@ -39,6 +39,7 @@ const renderScores = () => {
     data.result.sort((a, b) => b.score - a.score).forEach((element) => {
       const li = document.createElement('li');
```

Comment:
- [ ] [OPTIONAL] Kindly sort the scores, either alphabetically or numerically. This will ensure a better scoreboard or leaderboard viewing. check [this link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) to see the know how 😉.